packages/eslint-plugin/src/rules/padding-line-between-statements.ts

```diff
@@ -0,0 +1,720 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import {
+  RuleContext,
+  SourceCode,
+} from '@typescript-eslint/experimental-utils/dist/ts-eslint';
```

Comment:
do not deep import from this module.
Instead import the `TSESLint` namespace from the module and use that.

Reply:
I've just made this changes.